package.json

```diff
@@ -5,9 +5,9 @@
   "description": "We're building the next generation of Firefox, and we're focusing on real-world speed and performance. Pulse lets you give our engineers feedback about your experience on websites that work well in Firefox and on websites that don't.",
   "author": "Firefox Test Pilot",
   "license": "MPL-2.0",
-  "version": "1.1.2",
+  "version": "1.1.3",
   "homepage": "https://testpilot.firefox.com/experiments/pulse",
-  "hasEmbeddedWebExtension": true,
+  "hasEmbeddedWebExtension": false,
```

Comment:
Thought I'd read somewhere that flipping this bit after installation was problematic. But, I can't find a source for that, so I could just be confused.